app/locale/pl_PL/Sendit_Bliskapaczka.csv

```diff
@@ -31,6 +31,8 @@
 "number","Numer"
 "status","Status"
 "delivery_type","Typ przesyłki"
+"first name","Imię"
```

Comment:
Jestem praktycznie pewien, że magento ma już tłumaczenie 'first name'. Może jest w innej formie, np: First name. Tłumaczenia zaciągamy z https://github.com/SnowdogApps/Magento-Translation-pl_PL/archive/master.zip. Jak coś to zawsze to miejsce znajdziesz w pliku dockerfile.

Reply:
Dzięki za radę. Znalazłem tłumaczenie